### PR TITLE
Home Screen Cards Alignment

### LINKED ISSUE #501: Home screen alignment needs fixing
* **I'm submitting a ...**
  - [x] bug report
  - [ ] feature request

* **What is the current status?**
- The home screen alignment uses centre alignment for an odd number of cards.

* **If the current behaviour is a bug, please provide the steps to reproduce it**

The robotic arm card is centred and looks weird. 

* **What is the expected behaviour?**
- Left alignment should be used for the odd-numbered card.

* **Please tell us about your environment:**
  - OS with version number: Ubuntu 18.04

* **Would you like to work on it?**
Yes

### PR DESCRIPTION
**Pull request by priyanshunayan**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [x] Bug fix
- [ ] Feature implementation
- [ ] Doc updates

Fixes #501 
* **What changes have you introduced?**
I have made the cards align to the center in case of even cards in a row but aligned to start if the number is odd. 

* **Does this PR introduce a breaking change?**
No.

* **Preview / Steps to verify your work**:
image

### CODE REVIEW

File: src/screen/Home/components/Tabs.styles.js [hunk 1]
```diff
@@ -10,10 +10,14 @@ export const TabsContainer = styled.div`
...
   display: flex;
+  justify-content: flex-start;
+  align-items: center;
 `;
-
 export const TabsRow = styled.div`
   display: flex;
   justify-content: center;
+  align-items: center;
+  align-items: center;
```
Comment (CloudyPadmal) [inline src/screen/Home/components/Tabs.styles.js]: Why two?

Comment (priyanshunayan) [reply src/screen/Home/components/Tabs.styles.js]: I am so sorry. It is by mistake. I am correcting it right now.

Comment (CloudyPadmal) [reply src/screen/Home/components/Tabs.styles.js]: No worries. Thanks.

**Comment by AakashMallik** [comment]
@priyanshunayan 


On a high res screen, the whole thing is getting left indented... The code is not exactly solving the issue. Please check again.

**Comment by priyanshunayan** [comment]
Now it looks fine on both screen. 

image

**Pull request merged**